index.html

```diff
@@ -51,119 +51,121 @@ <h2>Release Calendar</h2>
-						<li><a href="/releases/mainnet/1.3.0/index.html">Release 1.3.0</a> (Monday, 19.04.2021)</li>
-						<li><a href="/releases/mainnet/1.2.11/index.html">Release 1.2.11</a> (Friday, 09.04.2021)</li>
-						<li><a href="/releases/mainnet/1.2.10/index.html">Release 1.2.10</a> (Friday, 12.03.2021)</li>
-						<li><a href="/releases/mainnet/1.2.9/index.html">Release 1.2.9</a> (Friday, 05.03.2021)</li>
-						<li><a href="/releases/mainnet/1.2.8/index.html">Release 1.2.8</a> (Friday, 26.02.2021)</li>
-						<li><a href="/releases/mainnet/1.2.7/index.html">Release 1.2.7</a> (Friday, 19.02.2021)</li>
-						<li><a href="/releases/mainnet/1.2.6/index.html">Release 1.2.6</a> (Friday, 12.02.2021)</li>
-						<li><a href="/releases/mainnet/1.2.5/index.html">Release 1.2.5</a> (Friday, 05.02.2021)</li>
-						<li><a href="/releases/mainnet/1.2.4/index.html">Release 1.2.4</a> (Friday, 22.01.2021)</li>
-						<li><a href="/releases/mainnet/1.2.2/index.html">Release 1.2.2</a> (Friday, 18.12.2020)</li>
-						<li><a href="/releases/mainnet/1.2.1/index.html">Release 1.2.1</a> (Friday, 11.12.2020)</li>
-						<li><a href="/releases/mainnet/1.2.0/index.html">Release 1.2.0</a> (Wednesday, 25.11.2020)</li>
-						<li><a href="/releases/mainnet/1.1.5/index.html">Release 1.1.5</a> (Friday, 13.11.2020)</li>
-						<li><a href="/releases/mainnet/1.1.4/index.html">Release 1.1.4</a> (Friday, 06.11.2020)</li>
-						<li><a href="/releases/mainnet/1.1.3/index.html">Release 1.1.3</a> (Friday, 16.10.2020)</li>
-						<li><a href="/releases/mainnet/1.1.2/index.html">Release 1.1.2</a> (Monday, 28.09.2020)</li>
-						<li><a href="/releases/mainnet/1.1.1/index.html">Release 1.1.1</a> (Friday, 04.09.2020)</li>
-						<li><a href="/releases/mainnet/1.1.0/index.html">Release 1.1.0</a> (Thursday, 27.08.2020)</li>
-						<li><a href="/releases/mainnet/1.0.9/index.html">Release 1.0.9</a> (Thursday, 23.07.2020)</li>
-						<li><a href="/releases/mainnet/1.0.8/index.html">Release 1.0.8</a> (Thursday, 25.06.2020)</li>
-						<li><a href="/releases/mainnet/1.0.7/index.html">Release 1.0.7</a> (Thursday, 04.06.2020)</li>
-						<li><a href="/releases/mainnet/1.0.6/index.html">Release 1.0.6</a> (Friday, 01.05.2020)</li>
-						<li><a href="/releases/mainnet/1.0.5/index.html">Release 1.0.5</a> (Friday, 17.04.2020)</li>
-						<li><a href="/releases/mainnet/1.0.4/index.html">Release 1.0.4</a> (Wednesday, 18.03.2020)</li>
-						<li><a href="/releases/mainnet/1.0.3/index.html">Release 1.0.3</a> (Thursday, 27.02.2020)</li>
-						<li><a href="/releases/mainnet/1.0.2/index.html">Release 1.0.2</a> (Tuesday, 14.01.2020)</li>
-						<li><a href="/releases/mainnet/1.0.1/index.html">Release 1.0.1</a> (Tuesday, 03.12.2019)</li>
-						<li><a href="/releases/mainnet/1.0.0/index.html">Release 1.0.0</a> (Monday, 18.11.2019) - <b>Identical to 0.2.17 Kovan release</b></li>
+						<li><a href="/releases/mainnet/1.9.2/index.html">Release 1.9.2</a> (Friday, 2021.07.23)</li>
```

Comment:
This date format thing was just discussed in the forums and LFW mentioned like the preferred one was YYYY-mm-dd (speaking about the forums).
I think it would be better to use that delimiter as it is the most used one for the international format.